modules/dxp/apps/commerce/commerce-machine-learning-test/src/testIntegration/java/com/liferay/commerce/machine/learning/recommendation/test/FrequentPatternCommerceMLRecommendationManagerTest.java

```diff
@@ -84,26 +82,28 @@ public void testGetFrequentPatternCommerceMLRecommendations()
 
 		long[] antecedentIds = ArrayUtil.toLongArray(antecedentIdList);
 
-		Stream<FrequentPatternCommerceMLRecommendation>
-			frequentPatternCommerceMLRecommendationStream =
-				_frequentPatternCommerceMLRecommendations.stream();
+		List<FrequentPatternCommerceMLRecommendation>
+			expectedFrequentPatternCommerceMLRecommendations = ListUtil.filter(
+				_frequentPatternCommerceMLRecommendations,
+				recommendation ->
+					_filterFrequentPatternCommerceMLRecommendation(
+						recommendation, antecedentIds));
+
+		Collections.sort(
```

Comment:
@jeffwu0724 IntelliJ recommends to use `List::sort`. Please change, thanks.

Reply:
Thanks Dante. Is this because if we call List::sort, it will just do it directly. If we call Colleciotns.sort(), it will call List::sort to do the job? 
Thanks!

Reply:
@jeffwu0724 It seems to be a matter of the implementation of `Collections.sort()`. The implementation is different in different JDK versions, and it delegates to `List.sort()` only after a certain version of JDK.